types.go

```diff
@@ -3,29 +3,13 @@ package sectorbuilder
 import (
 	"sync"
 
-	ffi "github.com/filecoin-project/filecoin-ffi"
 	"github.com/filecoin-project/go-address"
 	"github.com/filecoin-project/specs-actors/actors/abi"
 	"github.com/ipfs/go-datastore"
 
 	"github.com/filecoin-project/go-sectorbuilder/fs"
 )
 
-type SortedPublicSectorInfo = ffi.SortedPublicSectorInfo
```

Comment:
I know this was a specific ask, and it's not great that these aliases fail to give us the compile time guarantees we'd like, but these aliases do give us a bit of abstraction we could end up missing. We could seamlessly convert them to structs to shield implementations from small changes to the ffi layer that can be handled in sector builder. 

It's a small thing though. I'm not advocating for a revert.